code/modules/mob/living/carbon/human/life.dm

```diff
@@ -131,10 +131,14 @@
 	if((fire_stacks > HUMAN_FIRE_STACK_ICON_NUM && last_fire_update <= HUMAN_FIRE_STACK_ICON_NUM) || (fire_stacks <= HUMAN_FIRE_STACK_ICON_NUM && last_fire_update > HUMAN_FIRE_STACK_ICON_NUM))
 		last_fire_update = fire_stacks
 		update_fire()
-
 	..()
 	if(dna)
 		dna.species.handle_fire(src)
+	. = ..()
+	if(mind)
```

Comment:
less of an issue than the above but this should be put in the antag datum's process/whatever rather than here if possible

Reply:
Moved it back

Reply:
I moved it there because Ghommie in my previous PR requested it be moved there.